test/rubygems/test_gem_command_manager.rb

```diff
@@ -78,7 +78,7 @@ def test_find_command_unknown_suggestions
 
     message = "Unknown command pish".dup
 
-    if defined?(DidYouMean::SPELL_CHECKERS) && defined?(DidYouMean::Correctable)
+    if defined?(DidYouMean)
       message << "\nDid you mean?  \"push\""
```

Comment:
This fails randomly on ruby/ruby's CI https://github.com/ruby/ruby/actions/runs/17502806271/job/49719493474. Do you have any idea how we can stabilize this test?

Reply:
Almost every commit/PR is failing this test in one of the jobs in the workflows, which makes ruby/ruby PRs un-mergeable. I turned off the test for ruby/ruby for now https://github.com/ruby/ruby/commit/cd07c3cbae7e287350d713ead237aeef27cc2b9e.

Reply:
Apologies for the trouble. I think @nobu fixed the root cause with https://github.com/ruby/ruby/commit/ac24f70fb0c0cc80d30ec6e96b3369079775d0dc.